cli/cmd/generate_gcp.go

```
@@ -0,0 +1,780 @@
+		"bucket_region",
+		"",
+		"specify gcp bucket region")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.BucketLocation,
+		"bucket_location",
+		"",
+		"specify gcp bucket location")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.BucketName,
+		"bucket_name",
+		"",
+		"specify new bucket name")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.ExistingLogBucketName,
+		"existing_bucket_name",
+		"",
+		"specify existing bucket name")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.ExistingLogSinkName,
+		"existing_sink_name",
+		"",
+		"specify existing sink name")
+	generateGcpTfCommand.PersistentFlags().BoolVar(
+		&GenerateGcpCommandState.EnableForceDestroyBucket,
+		"enable_force_destroy_bucket",
+		false,
+		"enable force bucket destroy")
+	generateGcpTfCommand.PersistentFlags().BoolVar(
+		&GenerateGcpCommandState.EnableUBLA,
+		"enable_ubla",
+		false,
+		"enable universal bucket level access(ubla)")
+	generateGcpTfCommand.PersistentFlags().IntVar(
+		&GenerateGcpCommandState.LogBucketLifecycleRuleAge,
+		"bucket_lifecycle_rule_age",
+		-1,
+		"specify the lifecycle rule age")
+	generateGcpTfCommand.PersistentFlags().IntVar(
+		&GenerateGcpCommandState.LogBucketRetentionDays,
+		"bucket_retention_days",
+		0,
+		"specify the bucket retention days")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.AuditLogIntegrationName,
+		"audit_log_integration_name",
+		"",
+		"specify a custom audit log integration name")
+
+}
+
+func validateServiceAccountCredentialsFile(credFile string) error {
+	if file.FileExists(credFile) {
+		jsonFile, err := os.Open(credFile)
+		// if we os.Open returns an error then handle it
+		if err != nil {
+			return errors.Wrap(err, "Issue opening GCP credentials file")
+		}
+		defer jsonFile.Close()
+
+		byteValue, _ := ioutil.ReadAll(jsonFile)
```

Why are we ignoring this error?